lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,42 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
+        Optional output array for result.
+
+    re1: number
+        Real component.
+
+    im1: number
+        Imaginary component.
+
+    re2: number
+        Real component.
+
+    im2: number
+        Imaginary component.
+
+    Returns
+    -------
+    out: Array|TypedArray|Object
+        Real and imaginary components.
+
+    Examples
+    --------
+    > var y = {{alias}}( -13.0, -1.0, -2.0, 1.0 )
+    [ 5.0, 3.0 ]
+
+    > var out = new Float64Array( 2 );
+
+    > var v = {{alias}}( out, -13.0, -1.0, -2.0, 1.0 );
```

Comment:
In REPL text, semicolons have significance in the same way that semicolons in MATLAB have significance. Their presence silences output, and the absence allows output to be printed. So here, you want

```
> var out = new Float64Array( 2 );
> v = {{alias}}( out, -13.0, -1.0, -2.0, 1.0 )
<Float64Array>[ 5.0, 3.0 ]
> var bool = ( v === out )
true
```